Configure

```diff
@@ -581,7 +581,7 @@ my @disable_cascades = (
 
     "module"            => [ "fips", "dso" ],
 
-    "engine"            => [ grep /eng$/, @disablables ],
+    "engine"            => [ "dynamic-engine", "static-engine", grep(/eng$/, @disablables) ],
```

Comment:
No!

`static-engine` is handled further down in Configure, as a special case.  `%disabled` shall *only* have `dynamic-engine`, or not.  The rest is handled properly here:

https://github.com/openssl/openssl/blob/2d840893e78253bcce428603fdbcda159bdebe08/Configure#L1454-L1458

and here:

https://github.com/openssl/openssl/blob/2d840893e78253bcce428603fdbcda159bdebe08/Configure#L1784-L1788